actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BulkUploadManagementActor.java

```diff
@@ -48,10 +49,13 @@
   private int batchDataSize = 0;
   BulkUploadProcessTaskDao bulkUploadProcessTaskDao = new BulkUploadProcessTaskDaoImpl();
   private ObjectMapper mapper = new ObjectMapper();
+  private String OPERATION_PROGRESS_MSG = "Operation is {0}.";
```

Comment:
Add these constants in BulkUploadConstants.java.

Rename OPERATION_PROGRESS_MSG to OPERATION_STATUS_MSG.